src/displays/display-distance-slider.html

```diff
@@ -210,7 +212,17 @@
                         }
                     });
             }
-
+            _routeChanger() {
```

Comment:
_routeChanger sounds like this function will change the route, can you rename this to _routeChanged